source/_components/sensor.sht31.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Sensirion SHT31 Sensor"
+description: "Instructions how to integrate SHT31 sensors within Home Assistant."
+date: 2018-03-06 19:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: DIY
+ha_release: N/A
+logo: dht.png
```

Comment:
I don't think this is a DHT sensor 😬

Reply:
Other than this markdown file and a new image, is there anything else I should add for a new brand/sensor? Thanks